examples/nlp/neural_machine_translation_with_keras_nlp.py

```diff
@@ -146,18 +149,11 @@
 
```

Comment:
Same here, above this line, replace the "Luckily" sentence with the suggestion I had above or something like it.